openpype/hosts/resolve/api/workio.py

```diff
@@ -82,11 +85,9 @@ def current_file():
     current_dir = os.getenv("AVALON_WORKDIR")
     project = pm.GetCurrentProject()
     name = project.GetName()
-    fname = name + exported_projet_ext
+    fname = name + exported_project_ext
     current_file = os.path.join(current_dir, fname)
-    if not current_file:
-        return None
-    return os.path.normpath(current_file)
+    return os.path.normpath(current_file) if current_file else None
```

Comment:
This is the same as:
```suggestion
    if current_file:
        return os.path.normpath(current_file)
```
Without the need to mention `None` explicitly. Not sure if better, just wanted to throw it out there.